src/AvaloniaEdit/Editing/CaretNavigationCommandHandler.cs

```diff
@@ -108,7 +108,7 @@ static CaretNavigationCommandHandler()
             AddBinding(EditingCommands.MoveToDocumentEnd, InputModifiers.Control, Key.End, OnMoveCaret(CaretMovementType.DocumentEnd));
             AddBinding(EditingCommands.SelectToDocumentEnd, InputModifiers.Control | InputModifiers.Shift, Key.End, OnMoveCaretExtendSelection(CaretMovementType.DocumentEnd));
 
-            AddBinding(ApplicationCommands.SelectAll, InputModifiers.Control, Key.A, OnSelectAll);
+            AddBinding(ApplicationCommands.SelectAll, InputModifiers.None, Key.None, OnSelectAll);
```

Comment:
Can you explain this change?

Reply:
Sorry for really late response. 
It looked like some kind of workaround that key combinations for all ApplicationCommands was declared in class but for "SelectAll" doesn't. So I moved declaration to ApplicationCommands

Reply:
Then I don't think this line is needed any more.

Reply:
It's needed to add command binding. 

I created overload of AddBinding method which doesn't need those extra arguments.